packages/core/test/ignition-module-serializer.ts

```diff
@@ -90,6 +90,7 @@ describe("stored deployment serializer", () => {
     const fakeArtifact: Artifact = {
```

Comment:
We should probably share up `fakeArtifact` into the test helpers? What do you think?

Reply:
After checking in the helpers, it turns out that `fakeArtifact` already exists as a helper, but we're not using it. I added a commit propagating that across the tests.